src/Generic.Repository/Cache/CacheRepositoryFacade.cs

```diff
@@ -9,18 +9,20 @@ namespace Generic.Repository.Cache
 {
     internal class CacheRepositoryFacade : ICacheRepositoryFacade
     {
+        private readonly object _delegateLock = new object();
+
         public Action<object, object> CreateAction<TValue>(PropertyInfo property)
         {
             ThrowErrorIf.
-                IsNullValue(property, nameof(property), nameof(ExtractMethod));
+                IsNullValue(property, nameof(property), nameof(CreateAction));
 
             var setter = property.
                 GetSetMethod(true);
 
             ThrowErrorIf.
-                IsNullValue(setter, nameof(setter), nameof(ExtractMethod)); ;
+                IsNullValue(setter, nameof(setter), nameof(CreateAction)); ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/guilhermecaixeta/Generic.Service.DotNetCore2.2/pullRequest?prid=4961311)